_SUMMARY.md

```diff
@@ -186,6 +186,8 @@
     * [Implementing Parser and PSI](reference_guide/custom_language_support/implementing_parser_and_psi.md)
     * [Syntax Highlighting and Error Highlighting](reference_guide/custom_language_support/syntax_highlighting_and_error_highlighting.md)
     * [References and Resolve](reference_guide/custom_language_support/references_and_resolve.md)
+    * [Symbols (2020.3)](reference_guide/custom_language_support/symbols_20203.md)
+    * [Declarations and References (2020.3)](reference_guide/custom_language_support/declarations_and_references_20203.md)
```

Comment:
is "Declarations and References"/"Navigation" supposed to be _under_ "Symbols" node?

Reply:
I'd rather not put "(2020.3)" in menu but add note on top of each page instead

Reply:
> is "Declarations and References"/"Navigation" supposed to be under "Symbols" node?

No, why?

> I'd rather not put "(2020.3)" in menu but add note on top of each page instead

"WARNING This API is available starting from 2020.3 and currently in development and thus in experimental state" ?